src/components/popup-dialog/PopupDialog.js

```diff
@@ -6,7 +6,7 @@ import useConfirm from '~/hooks/use-confirm'
 import useBreakpoints from '~/hooks/use-breakpoints'
 import { style } from '~/components/popup-dialog/popup-dialog.style'
 
-const PopupDialog = ({ content, closeModal, isFullScreen, setFullScreen }) => {
+const PopupDialog = ({ content, closeModal, isFullScreen, paperProps, setPaperProps, setFullScreen }) => {
```

Comment:
paperProps = {}

Reply:
it has to be state so we can set props to context